mainnet-rehearsals/3-R4-upgrade-2-of-2/records/1/deployment-output.json

```diff
@@ -0,0 +1,5 @@
+{
+  "L1ERC721BridgeProxy": "0xce479c1299AdBE12f34A479F404937151AF19D92",
+  "OptimismPortalProxy": "0xDd3840F548Bd8fA41421050ed7C28753B494734c",
+  "ProxyAdmin": "0xEA91B6491c4c8E6168D96176005Cea0133fD0FC0"
+}
```

Comment:
we should verify these addresses before the rehearsal, as it should make the tenderly simulation more readable.